packages/components/pie-toast/src/defs.ts

```diff
@@ -44,6 +46,13 @@ export interface ToastProps {
    * The leading action for the toast.
    */
   leadingAction?: ActionProps;
+
+  /**
+   * It set the duration of the toast in milliseconds before it auto-dismiss
+   * If the value is null auto-dismiss is disabled
+   * If the value is not provided it auto-dismiss after 5 seconds (5000 milliseconds)
```

Comment:
`If the value is not provided it auto-dismisses after 5 seconds (5000 milliseconds)`